server/services/database/documents.js

```diff
@@ -48,7 +48,10 @@ const query =
       inner join languages as lt\
       on lt.code = d.to_language_code\
       inner join users as u\
-      on u.id = d.owner_id";
+      on u.id = d.owner_id\
+      left join translations t\
+      on t.document_id = d.id\
+      ";
```

Comment:
You need to return also t.id, it is currently missing.

Reply:
I don't think I should have changed this code at first place. These lines of code were changed in pull request 35 as well, so I pulled changes from upstream master.  I got conflict and i resolved it by discarding my changes, so now should this be as it was before.